npe2/manifest/_validators.py

```diff
@@ -60,8 +60,7 @@ def python_name(name: str) -> str:
 def display_name(v: str) -> str:
     if not DISPLAY_NAME_PATTERN.match(v):
         raise ValueError(
-            f"{v} is not a valid display_name.  The display_name must "
-            "be 3-40 characters long, containing printable word characters, "
+            f"{v} is not a valid display_name. It must be 3-40 characters long, "
```

Comment:
```suggestion
            f"{v} is not a valid display_name. It must be 3-90 characters long, "
```

Reply:
lol.  I corrected that like 3 times and _still_ ended up with 40 :joy:  thank you